original/v1/s006-lists/handout-ostrava/handout6.tex

```diff
@@ -88,9 +88,9 @@ \section*{Domácí projekty \plsetno}
     \\Domácí zvířata známe tato: \verb+"pes", "kočka", "králík", "had"+.
 
 
-\item Napiš funkci, která vypíše jména domácích zvířat, která jsou kratší než 5 písmen.
+\item Napiš funkci, která vrací jména zadaných domácích zvířat, která jsou kratší než 5 písmen.
```

Comment:
@encukou taky jsem použil slovo „zadaných“ (dělali nám funkce, co pracují s globálním seznamem, moc se nám to s @messa nelíbilo)